x/staking/keeper/delegation_test.go

```diff
@@ -665,7 +665,7 @@ func (s *KeeperTestSuite) TestUnbondingAllDelegationFromValidator() {
 
 	// validator should now be deleted from state
 	_, err = keeper.GetValidator(ctx, addrVals[0])
-	require.ErrorIs(err, stakingtypes.ErrNoValidatorFound)
```

Comment:
what is the reason for changing the error?

Reply:
bcz we are no longer returning the `stakingtypes.ErrNoValidatorFound)` but returning `collections.ErrNotFound` from `GetValidator`